### DISCUSSION TITLE
Independent Qtile bar

### DISCUSSION THREAD

**Discussion by cr1ogen** [qtile/qtile#2140] [Ideas]
Good afternoon. Watching a video of Xmodbar, it occurred to me, if not an independent Qtile bar could be made, which could be called Qbar for example. Thanks so much for reading

**Comment by ramnes** [comment]
Yeah it could, and I would personally like it, but it would require quite some work and I'm not sure if we have someone willing to do it at the moment. Depending on what other developers think, we could merge pull requests that make the codebase move in that direction.

**Comment by m-col** [comment]
I like this idea primarily because I imagine the code would be much more modular and organised if it was implemented this way.
